pkgs/applications/version-management/git-backdate/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, git, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "git-backdate";
+  version = "2023-07-19";
+
+  src = fetchFromGitHub {
+    owner = "rixx";
+    repo = pname;
+    rev = "8ba5a0eba04e5559be2e4b1b6e02e62b64ca4dd8";
+    sha256 = "sha256-91cEGQ0FtoiHEZHQ93jPFHF2vLoeQuBidykePFHtrsY=";
+  };
+
+  nativeBuildInputs = [
```

this way it gets automatically `patchShebangs`ed
```suggestion
  buildInputs = [
```